services/datalake/pod-datalake/src/handlers/image.ts

```diff
@@ -80,31 +80,68 @@ export async function handleImageGet (
   const { workspace, name, transform } = req.params
 
   const accept = req.headers.accept ?? 'image/*'
-  const image = parseImageTransform(accept, transform)
+  const { format, width, height, fit } = getImageTransformParams(accept, transform)
+
+  const tempDir = mkdtempSync(join(tmpdir(), 'image-'))
```

Comment:
Bad-bad-bad, please create a temp dir on service init and use it. And clean by timer.